features/your_details_section.feature

```diff
@@ -15,4 +15,4 @@ Scenario: Display minimum contribution details
 Scenario: Display full pay contribution details
   And I click on "My employer makes contributions on all of my salary"
   And I click the Next button
-  Then I should see in English/Welsh my age, gender, salary, frequency and full pay
\ No newline at end of file
+  Then I should see in English/Welsh my age, gender, salary, frequency and full pay
```

Comment:
This step definition is not accurate. 

`Then I should see in English/Welsh my age, gender, salary, frequency and full` --> `Then I should see in English my age, gender, salary, frequency and full`

This is because the Welsh page is not being tested.